lambdas/service/app.py

```diff
@@ -1281,14 +1287,18 @@ def get_summary():
                                              authentication=request.authentication)
 
 
-def manifest_route(*, fetch: bool, initiate: bool):
+def manifest_route(*, fetch: bool, method: str):
```

Comment:
I don't like the proliferation of the string 'POST' that this method signature change causes.

Please try

```suggestion
def manifest_route(*, fetch: bool, initiate:bool, curl:bool):
```

in a separate fixup, after applying the other changes from my patch.